src/components/Collection/CollectionList/RenameCollectionAction.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+import useTranslation from 'next-translate/useTranslation';
+
+import RenameCollectionModal from '../RenameCollectionModal/RenameCollectionModal';
+
+import PopoverMenu from 'src/components/dls/PopoverMenu/PopoverMenu';
+import { updateCollection } from 'src/utils/auth/api';
+
+const RenameCollectionAction = ({ currentCollectionName, collectionId, onDone }) => {
+  const { t } = useTranslation();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const onMenuClicked = () => {
+    setIsModalOpen(true);
+  };
+
+  const onSubmit = (data) => {
+    updateCollection(collectionId, { name: data.name }).then(() => {
+      onDone();
+      setIsModalOpen(false);
+    });
```

Comment:
What happens when there is an issue with the API? I think we should show a toast/error message